attendance.py

```diff
@@ -0,0 +1,18 @@
+from student import Student
+
+class Attendance:
+    def __init__(self):
+        self.students = [
+            Student('1', 'John Doe'),
+            Student('2', 'Jane Doe'),
+            Student('3', 'Jim Doe')
+        ]
+
+    def get_students(self):
+        return self.students
+
+    def mark_attendance(self, student_id):
+        for student in self.students:
+            if student.id == student_id:
+                student.mark_attendance()
```

Comment:
Consider using a dictionary with 'student_id' as the key instead of a list for 'students'. This would improve the time complexity of the 'mark_attendance' function from O(n) to O(1).